qualtrics-event-gw/README.md

```diff
@@ -0,0 +1,93 @@
+# qualtrics-event-gw
+
```

Comment:
Please add a summary on what that component is about, main responsibilities

Reply:
done